RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Releasing
+
+Before releasing a new version of packages to the NPM and GitHub, Its advisable to follow the below steps in order
+
+- Update the dependencies `npm run clean`
+- Bootstrap the deps using `npm run bootstrap`
+- Make sure all tests passes `npm run test`
+
+### Releasing all packages
+
+To publish all the packages run `npm run publish`. It will run lerna publish (we use --independent) so it will prompt the version number for
+every package. `lerna publish` will take care of the following
+
+- Creating annonated Git tags for all the packages that has changed since last release
+- Generating `CHANGELOG.md` using conventional commit messages.
+- Publishing the packages via `npm publish`
+
+### Releasing single package
+
+To publish a single package, run `npm run release-package -- elastic-apm-js-base` which uses `lerna publish` under the hood to publish a single package to NPM
+
+### Releasing artifacts to GitHub
+
+Use `npm run release-github` in the root directory, the script takes care of building the `elastic-apm-js-base` files and creating a release in GitHub with the previous annonated tag.
```

Comment:
I think we should have one script in the package.json that does all of this including doing the tests and other checks

Reply:
Already covered it here, https://github.com/elastic/apm-agent-js-base/pull/166/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R11

so it will be `release-github: "npm run build && npm run test && node scripts/release-gihtub.js"`